mmv1/products/apphub/Application.yaml

```diff
@@ -0,0 +1,199 @@
+base_url: projects/{{project}}/locations/{{location}}/applications
+create_url: projects/{{project}}/locations/{{location}}/applications?applicationId={{application_id}}
+self_link: projects/{{project}}/locations/{{location}}/applications/{{application_id}}
+id_format: projects/{{project}}/locations/{{location}}/applications/{{application_id}}
+import_format:
+  - projects/{{project}}/locations/{{location}}/applications/{{application_id}}
+name: Application
+description: Description
+autogen_async: true
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: "application_basic"
+    primary_resource_id: "example"
+    config_path: "templates/terraform/examples/apphub_application_basic.tf.erb"
+    skip_test: true
+    vars:
+      location: "us-east1"
+      application_id: "example-application"
+      scope_type: "REGIONAL"
+  - !ruby/object:Provider::Terraform::Examples
+    name: "application_basic"
+    primary_resource_id: "example"
+    config_path: "templates/terraform/examples/apphub_application_basic.tf.erb"
+    skip_docs: true
+    vars:
+      application_id: "example-application"
+  - !ruby/object:Provider::Terraform::Examples
+    name: "application_full"
+    primary_resource_id: "example2"
+    config_path: "templates/terraform/examples/apphub_application_full.tf.erb"
+    vars:
+      application_id: "example-application"
+      display_name: "Application Full"
+      description: "Application for testing"
+      business_name: "Alice"
+      business_email: "alice@google.com"
+      developer_name: "Bob"
+      developer_email: "bob@google.com"
+      operator_name: "Charlie"
+      operator_email: "charlie@google.com"
+properties:
+  - !ruby/object:Api::Type::String
+    name: name
+    output: true
+    description: "Identifier. The resource name of an Application. Format:\n\"projects/{host-project-id}/locations/{location}/applications/{application-id}\" "
+  - !ruby/object:Api::Type::String
+    name: displayName
+    description: 'Optional. User-defined name for the Application. '
+  - !ruby/object:Api::Type::String
+    name: description
+    description: 'Optional. User-defined description of an Application. '
+  - !ruby/object:Api::Type::NestedObject
+    properties:
+      - !ruby/object:Api::Type::NestedObject
+        properties:
+          - !ruby/object:Api::Type::Enum
+            name: type
+            description: 'Criticality type. '
+            required: true
+            values:
+              - :TYPE_UNSPECIFIED
+              - :MISSION_CRITICAL
+              - :HIGH
+              - :MEDIUM
+              - :LOW
+        name: criticality
+        description: 'Criticality of the Application, Service, or Workload '
+      - !ruby/object:Api::Type::NestedObject
+        properties:
+          - !ruby/object:Api::Type::Enum
+            name: type
+            description: 'Environment type. '
+            required: true
+            values:
+              - :TYPE_UNSPECIFIED
```

Comment:
Please remove TYPE_UNSPECIFIED from these enums. It's almost never a valid value for a customer to specify

Reply:
Done